app/views/flood_risk_engine/shared/_postcode.html.erb

```diff
@@ -24,3 +24,21 @@
   <% end  %>
 
 </fieldset>
+
+<div class="form-group">
+  <%= f.submit(step_t(step, ".continue",  default: t("global.continue")), class: "button") %>
+</div>
+&nbsp;
+<hr />
+<p class="text font-xsmall" id="os-notice">
+  <%=
+    t(
+      ".os_notice.text",
+      link: link_to(
+        t(".os_notice.url_text"),
+        enrollment_page_path(form.enrollment, id: "os_places_terms"),
+        target: "_blank"
+      )
+    ).html_safe
```

Comment:
Rather than use `html_safe` it's worth noting you can use the i18n key name (end the key name with `_html`) to determine the ouptut of html in i18n translations:

http://guides.rubyonrails.org/i18n.html#using-safe-html-translations